quiche/src/lib.rs

```diff
@@ -1350,6 +1355,20 @@ impl Config {
     pub fn set_disable_dcid_reuse(&mut self, v: bool) {
         self.disable_dcid_reuse = v;
     }
+
+    /// Configures tracking unknown transport params.
+    ///
+    /// To enable, pass `Some` `usize` with the maximum
+    /// amount of space to expend to track the ids and
+    /// values of unknown transport parameters. To disable,
+    /// pass `None`.
+    ///
+    /// The default is that the feature is disabled.
+    pub fn enable_track_unknown_transport_parameters(
+        &mut self, size: Option<usize>,
```

Comment:
```suggestion
        &mut self, size: usize,
```

Since the default is off,  I don't see much value in allowing a caller to disable it. Therefore I think we can just make this a value without wrapping in an option, then mark it as Some(size) inside the function. This also makes it easier to deal with in the FFI, which is something this PR should also do (see quiche.g and ffi.rs).